suite-native/app/ios/TrezorSuite/Info.plist

```diff
@@ -70,5 +70,7 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>NSMicrophoneUsageDescription</key>
+    <string>This app does not require access to the microphone.</string>
```

Comment:
pls fix indetion :D

Reply:
Probably just issue on github as indentation seems fine in my IDE
![image](https://user-images.githubusercontent.com/36101761/222464709-207ef938-6e3a-4365-ab93-68cc5664b10c.png)

Reply:
It also looks fine on vs code in github if you open the web editor